cdap-client-tests/src/test/java/co/cask/cdap/client/MetricsClientTestRun.java

```diff
@@ -70,10 +71,18 @@ public void testAll() throws Exception {
       Id.Program programId = Id.Program.from(appId, ProgramType.FLOW, FakeFlow.NAME);
       String flowlet = FakeFlow.FLOWLET_NAME;
 
-      MetricQueryResult result = metricsClient.query(
-        ImmutableList.of(Constants.Metrics.Name.Flow.FLOWLET_INPUT),
-        ImmutableList.<String>of(),
-        MetricsTags.flowlet(programId, flowlet));
+      MetricQueryResult result = metricsClient.query(MetricsTags.flowlet(programId, flowlet),
+                                                     Constants.Metrics.Name.Flow.FLOWLET_INPUT);
+      Assert.assertEquals(1, result.getSeries()[0].getData()[0].getValue());
+
+      result = metricsClient.query(MetricsTags.flowlet(programId, flowlet),
+                                   ImmutableList.of(Constants.Metrics.Name.Flow.FLOWLET_INPUT),
+                                   ImmutableList.<String>of(), ImmutableMap.of("aggregate", "true"));
+      Assert.assertEquals(1, result.getSeries()[0].getData()[0].getValue());
```

Comment:
somewhere, we should have a utility like this:
https://github.com/caskdata/cdap-integration-tests/blob/develop/integration-test-remote/src/test/java/co/cask/cdap/apps/AudiTestBase.java#L136

Otherwise, accessing the aggregate value like this is messy.